pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -711,27 +719,26 @@ type HelperSpec struct {
 	ImagePullPolicy *corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
 }
 
-// +k8s:openapi-gen=true
 // TiDBSlowLogTailerSpec represents an optional log tailer sidecar with TiDB
+// +k8s:openapi-gen=true
 type TiDBSlowLogTailerSpec struct {
 	corev1.ResourceRequirements `json:",inline"`
 
-	// Image used for slowlog tailer
-	// Deprecated, use TidbCluster.HelperImage instead
+	// (Deprecated) Image used for slowlog tailer.
+	// Note that this is deprecated, use `TidbCluster.HelperImage` instead
 	// +k8s:openapi-gen=false
 	Image *string `json:"image,omitempty"`
 
-	// ImagePullPolicy of the component. Override the cluster-level imagePullPolicy if present
-	// Deprecated, use TidbCluster.HelperImagePullPolicy instead
+	// (Deprecated) ImagePullPolicy of the component. Override the cluster-level imagePullPolicy if present
+	// Note that this is deprecated, use `TidbCluster.HelperImagePullPolicy` instead
 	// +k8s:openapi-gen=false
 	ImagePullPolicy *corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
 }
 
-// +k8s:openapi-gen=true
 // ComponentSpec is the base spec of each component, the fields should always accessed by the Basic<Component>Spec() method to respect the cluster-level properties
+// +k8s:openapi-gen=true
 type ComponentSpec struct {
-	// Image of the component, override baseImage and version if present
-	// Deprecated
+	// (Deprecated) Image of the component, override baseImage and version if present
```

Comment:
This is incorrect.
Actually, this is useless now.

Reply:
Do we just delete `Image` field?

Reply:
This is for backward compatibility, we need to keep it.

Reply:
```suggestion
	// (Deprecated) Image of the component, use `baseImage` and `version`
```